src/System.Text.Json/src/System/Text/Json/Reader/Utf8JsonReader.cs

```diff
@@ -260,6 +267,102 @@ public bool Read()
             return retVal;
         }
 
+        /// <summary>
+        /// Skips the children of the current JSON token.
+        /// </summary>
+        /// <exception cref="InvalidOperationException">
```

Comment:
It also throws JsonReaderException (if that's still the type name) if the JSON is invalid... so anything that Read says it would throw.